### DISCUSSION TITLE
Template text content as gg.jte.Content

### DISCUSSION THREAD

**Discussion by knoppixmeister** [casid/jte#390] [Q&A]
Is it possible to put/convert whole template rendered content into gg.jte.Content object like it described here: https://jte.gg/syntax/#content ?

Now it only possible put content in some special syntax @\`...\` but not template as function (with params) call

**Comment by casid** [comment]
Is it something like this you would like to do?

https://github.com/casid/jte/issues/263#issuecomment-1696918913
